src/ResourceManager/Sql/Commands.Sql/Replication/Cmdlet/GetAzureSqlDatabaseReplicationLink.cs

```diff
@@ -70,14 +70,24 @@ protected override IEnumerable<AzureReplicationLinkModel> GetEntity()
         {
             ICollection<AzureReplicationLinkModel> results;
 
+            //if (ParameterSetName == ByPartnerServerName)
```

Comment:
Why is this all commented out?

Reply:
@akromm I forgot to remove these comment out code. the old code using Legacy database model to get/list replication links. I make the changes to use new autorest sdk database model to do the same operations.